### ISSUE TITLE
Duplicate queries

### ISSUE THREAD

**Issue by Alphabc** [elementor/elementor#9579]
## Prerequisites

- [x ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x ] The issue still exists against the latest stable version of Elementor.

I've sent this to the support team who asked me to open an issue here
Thx!

## Description

Hi, I'm using Elementor Pro along with Astra Pro and WooCommerce.
On the home page, and on many pages, there is a query that is marked as duplicate, from 4 to 9 times):
SELECT XXX_posts.*
FROM XXX_posts
WHERE 1=1
AND XXX_posts.post_type = 'elementor_icons'
AND ((XXX_posts.post_status = 'publish'))
ORDER BY XXX_posts.post_date DESC 9 wpdb->get_results
9 calls
Elementor\Icons_Manager::get_icon_manager_tabs_config
1 call
Elementor\Core\Files\CSS\Base->enqueue
1 call
Elementor\Frontend->maybe_enqueue_icon_font
2 calls
Elementor\Icons_Manager::render_icon_html
5 calls
![image](https://user-images.githubusercontent.com/24738945/68917724-8117af00-07a6-11ea-9b96-7a51f33a48d1.png)

is it that the query is within a loop?

## Isolating the problem

- [ x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a default WordPress theme active.
- [ x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
WP 5.2.4 + Elementor Pro, Astra Pro, Woocommerce all in last version
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by bainternet** [comment]
@Alphabc 

Thanks for reporting, should be fixed in next version.

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor Pro v2.8.1

Feel free to update

Thanks!

**Comment by abdulwahabaw** [comment]
Thanks allot my problem is solve
